File: gym_fe/src/types/schedule.ts
export interface Schedule {
  scheduleid: string;
  dayofweek: string;
  maxparticipants: number;
  starttime: string;
  endtime: string;
}

export interface ScheduleFormData {
  dayofweek: string;
  maxparticipants: number;
  starttime: string;
  endtime: string;
}

export interface ScheduleApiResponse {
  items: Schedule[];
  totalPages: number;
  currentPage: number;
  totalItems: number;
}

File: gym_be/gym_be/Models/Entities/Schedule.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace gym_be.Models.Entities
{
    [Table("schedule")]
    public class Schedule
    {
        [Key]
        [Column("scheduleid")]
        public Guid ScheduleID { get; set; }

        [Column("dayofweek")]
        public string DayOfWeek { get; set; }

        [Column("maxparticipants")]
        public int MaxParticipants { get; set; }

        [Column("starttime")]
        public DateTime StartTime { get; set; }

        [Column("endtime")]
        public DateTime EndTime { get; set; }

        public ICollection<Appointment> Appointments { get; set; }
    }
}
